### ISSUE TITLE
Brief introduction to prospect users

### ISSUE THREAD

**Issue by henkvancann** [henkvancann/MOOC-trials#27]
Aimed at those who are thinking about creating an account and / or enroll to a course.

<img width="887" alt="Screenshot 2022-04-12 at 12 09 47" src="https://user-images.githubusercontent.com/479356/162937145-29613032-8704-4d99-8c24-80f11efa8586.png">

## Original text to be shortened

MOOC-trials
 Cookies must be enabled in your browser (i)
 Some courses may allow guest access

Log in as a guest (button)

Is this your first time here?
Hi! For full access to courses you'll need to take
a minute to create a new account for yourself on this web site.
Each of the individual courses may also have a one-time
"enrolment key", which you can use during this sign up:
Fill out the [New Account](https://keri.blockchainbird.org/login/signup.php) form with your details.
You will be prompted for an "enrolment key" - use the one
that your teacher has given you. This will "enrol" you in the
course.

Your account will be created and you will be logged in.
You can now access the full course for this session.
An email has also been immediately sent to your email address.

Read your email, and click on the web link it contains.
From now on you will only need to enter your personal
username and password (in the form on this page) to log in
and access any course you have enrolled in.

**Comment by henkvancann** [comment]
### Proposed New Text

MOOC-trials
Cookies must be enabled in your browser (i)
Log in as a guest (button) (i)

First time here?
For full access to courses requires signup (i) and then enrolment (i)

Use **_Create New Account_** button. Please follow further instructions, that'll involve e-mail confirmation and lookup.


### Changes proposed

Some courses may allow guest access -> under an i - popup of Log in as a guest - button

Is this your first... shortened

add popup (i) after 'sign up' : you'll need to take
a minute to create a new account for yourself on this web site. 

add popup (i) after 'enrolment' : each of the individual courses may also have a one-time
"enrolment key"

Fill out the [New Account](https://keri.blockchainbird.org/login/signup.php) form with your details. -> Use Create New Account button.

Please follow further instructions, that'll involve e-mail confirmation and lookup. Use this information to explain further -> 

You will be prompted for an "enrolment key" - use the one
that your teacher has given you. This will "enrol" you in the
course.

Your account will be created and you will be logged in.
You can now access the full course for this session.
An email has also been immediately sent to your email address.

Read your email, and click on the web link it contains.
From now on you will only need to enter your personal
username and password (in the form on this page) to log in
and access any course you have enrolled in.

**Comment by ghost** [comment]
Hi Henk,

I'm sorry, but what you wrote is very unclear to me. I can't follow the lines and make sense of it. Could you please send me the whole text at once? Also, please note that it may be too complicated or impossible to add any additional buttons or i (information pop-ups). This is where the default text in Moodle is, and I need to add a custom text in that not-so-user-friendly box.
![image](https://user-images.githubusercontent.com/36804362/163351366-58be1753-900a-48e5-98e7-5da873bd1102.png)

**Comment by henkvancann** [comment]
We'll discuss it briefly next meeting. The text I propose is no more than this:

---

MOOC-trials
Cookies must be enabled in your browser (i)
Log in as a guest (button) (i)

First time here?
For full access to courses requires signup (i) and then enrolment (i)

Use Create New Account button. Please follow further instructions, that'll involve e-mail confirmation and lookup.

---
And if we can't add (i)- hover text, we'll just find a spot to explain.

**Comment by kordwarshuis** [comment]
Onboarding process at Slack discussed:
https://youtu.be/Uk4QyZhXC3w?t=434

**Comment by kordwarshuis** [comment]
Apparently changing the onboarding process is possible:

https://moodle.org/mod/forum/discuss.php?d=328529#p1322328

**Comment by ghost** [comment]
Yes, it can be done. But I was saying that I can't add (i) fields with additional information.
